soql-analyzer/src/test/scala/com/socrata/soql/SoQLAnalyzerTest.scala

```diff
@@ -506,13 +506,13 @@ SELECT visits, @x2.zx
     val analysisWordStyle = analyzer.analyzeUnchainedQuery("SELECT avg(visits) OVER (PARTITION BY name_last, 2, 3)")
     val select = analysisWordStyle.selection.toSeq
     select must equal (Seq(
-      ColumnName("wf_over_avg_visits_name_last_2_3") -> typedExpression("wf_over(avg(visits), name_last, 2, 3)")
+      ColumnName("wf_over_avg_visits_name_last_2_3") -> typedExpression("avg(visits) OVER (PARTITION BY name_last, 2, 3)")
     ))
 
     val analysisWordStyleOverEmpty = analyzer.analyzeUnchainedQuery("SELECT avg(visits) OVER ()")
     val selectOverEmpty = analysisWordStyleOverEmpty.selection.toSeq
     selectOverEmpty must equal (Seq(
-      ColumnName("wf_over_avg_visits") -> typedExpression("wf_over(avg(visits))")
+      ColumnName("wf_over_avg_visits") -> typedExpression("avg(visits) OVER ()")
```

Comment:
I just approved it, but I'm not sure this column name is the right thing to generate.  Can you see if it'll be plausible to generate avg_visits_over or the like?